presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -45,3 +45,100 @@ which is configured with a carefully selected set of options. This behavior
 can be overridden by setting the environment variable ``PRESTO_PAGER`` to the
 name of a different program such as ``more`` or `pspg <https://github.com/okbob/pspg>`_,
 or it can be set to an empty value to completely disable pagination.
+
+Output Formats
+--------------
+
+The Presto CLI provides the option ``--output-format`` to control how the output is
+displayed when running in noninteractive mode. The available options shown in the
+following table must be entered in uppercase. The default value is ``CSV``.
+
++---------------------+------------------------------------------------------------+
```

Comment:
Just a personal note on tables in sphinx ... the list-table syntax is less error prone for editing and reviews in my experience.